app/jobs/process_patient_changesets_job.rb

```diff
@@ -143,11 +143,7 @@ def search_for_patient(attrs, step_name)
   rescue NHS::PDS::TooManyMatches
     [:too_many_matches, nil]
   rescue Faraday::ClientError, Faraday::ServerError => e
-    Rails.logger.error(
-      "Error doing PDS search for patient changeset #{patient_changeset.id}: #{e.message}"
-    )
```

Comment:
Ugh. Sorry about this. However, I do think we should have logging here for Splunk and live services ... I say we log without the `patient_changeset` anyway. I thought I had added some logging context somewhere ... let me look

Reply:
Would it make sense to add a call to `SemanticLogger.tagged(patient_changeset_id: patient_changeset.id)` in the `perform`, like we did here: https://github.com/nhsuk/manage-vaccinations-in-schools/blob/main/app/jobs/sync_vaccination_record_to_nhs_job.rb#L13